src/Sylius/Component/Order/Model/OrderItemUnit.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Order\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class OrderItemUnit implements OrderItemUnitInterface
+{
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var int
+     */
+    protected $total = 0;
```

Comment:
What is `total` at the `OrderItemUnit` level?

At the `OrderItem` level it was (`quantity` * `unit_price`) + `adjustments_total`, which was calculated, likewise at the order level.

At the unit level what do we need here? Is there a `price` or `value` field that gets set based on the `OrderItem.unitPrice`? `total` would then be calculated based on `price` + `adjustmentsTotal`?